src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -0,0 +1,57 @@
+import _ from 'lodash/fp';
+
+import * as currentOrPastDate from '../../common/schemaform/definitions/currentOrPastDate';
+import * as fullName from '../../common/schemaform/definitions/fullName';
+import * as ssn from '../../common/schemaform/definitions/ssn';
+
+import definitions from 'vets-json-schema/dist/definitions.json';
+
+import { relationshipLabels, genderLabels } from '../utils/helpers';
+
+const defaultProps = [
+  'relativeFullName',
+  'relativeSocialSecurityNumber',
+  'relativeDateOfBirth',
+  'gender',
+  'relationship'
+];
+
+export default function applicantInformation(propNames = defaultProps) {
+  const possibleProperties = {
+    relativeFullName: fullName.schema,
+    relativeSocialSecurityNumber: ssn.schema,
+    relativeDateOfBirth: currentOrPastDate.schema,
+    gender: definitions.gender,
+    relationship: definitions.relationship
+  };
+
+  return {
+    path: 'applicant-information',
+    title: 'Applicant information',
+    initialData: {},
+    uiSchema: {
```

Comment:
I think it'd be good to put `'ui:order': propNames` in here to be safe. I'm not positive the order is based on the list of fields in `_.pick`, and if not it could change depending on the schema